content/tutorial/02-sveltekit/01-concepts/03-server-and-client/README.md

```diff
@@ -8,4 +8,6 @@ A SvelteKit app can be thought of as two distinct entities working in tandem —
 
 'Client' refers to the JavaScript that loads in the browser.
 
-TODO
+SvelteKit makes the two communicate with each other seamlessly. On the initial page load, the server renders the HTML, meaning content is visible as quickly as possible. The client then takes over in a process called 'hydration', so that subsequent navigations happen without full page reloads. It will request additional code and data from the server as needed.
```

Comment:
'server the server-side'... 😬 

The confusingness doesn't just come from serverless/edge functions, but also prerendering (and one day, service workers)

Reply:
> 'serve the server-side'...

haha. yeah, that was bad. updated to remove that

> The confusingness doesn't just come from serverless/edge functions, but also prerendering

It sounds like the confusing part isn't really the _word_ "server"? Maybe we should just swap the word "confusing" with "apropos"?

> and one day, service workers

In that case we'd have to also reword the prior sentence as there's no longer "two distinction entities", but now three: the client, server, and service worker.

Reply:
> Maybe we should just swap the word "confusing" with "apropos"?

These are essentially opposites?

Again, among other things we're talking about prerendering here. In a literal sense there is no server at all, even a local server — it's just a function. But conceptually that function is a server.

In the service worker case, there are still two entities; the service worker _is_ the 'server'.

If there's a better word than 'server' to describe the abstract concept of 'thing that takes a `Request` as input and returns a `Response` then I'm all ears, but I don't think such a thing exists. Absent one, our best approach is to use the common terminology but be upfront about its shortcomings.

Reply:
I should have been clearer. By swap, I didn't mean a 1:1 swap, but rewriting to make it work with that word. Like:

>'Server' is, perhaps, not the most apropos word since your app will often be running in a _serverless_ environment (cloud/edge functions) or might even be deployed as a set of completely static files. But it's the best we've got. The server's basic job is to turn a request into a response.

Reply:
I still think

> 'Server' is, perhaps, a confusing word

is better — 'not the most apropos word' implies that there's a _more_ apropos word, in which case why aren't we using it? The fact is that there _isn't_ a more apropos word. We should just acknowledge the fact and move on, I think